### DISCUSSION TITLE
cannot install in xampp

### DISCUSSION THREAD

**Discussion by mnmnnmnnmnmnmn** [givanz/Vvveb#22] [General]
xampp newest verision windows 10
php 8.2.0
apache2.4.54
openssl 1.1.1 (not being used)
php myadmin 5.2.0
mariadb 10.4.27
windows10
host : 127.0.0.1

after install, says that delete db.php

whats the problem?

**Comment by mnmnnmnnmnmnmn** [comment]
huh?? i dont know how i fixed. i tried for hours..

**Comment by kvirtu** [comment]
may not be supported php **8.2.0** , try 8.0

**Comment by givanz** [comment]
>after install, says that delete db.php whats the problem? 

config/db.php does not exist before installation it is created after install is finished.
Probably the config/db.php was created by a failed install, removing the file will allow reinstall.
The delete config/db.php message is shown if install is attempted while the file exists warning that the app is already installed.

> huh?? i dont know how i fixed. i tried for hours..

Probably you deleted config/db.php before retrying.

> may not be supported php 8.2.0 , try 8.0

Code is written on an environment with php 8.2.0 it should work best with this php version.
